crates/turborepo-lib/src/package_manager/mod.rs

```diff
@@ -433,6 +446,7 @@ impl PackageManager {
     pub fn lockfile_name(&self) -> &'static str {
         match self {
             PackageManager::Npm => npm::LOCKFILE,
+            PackageManager::Bun => bun::LOCKFILE,
```

Comment:
I think we need to be careful with this since `read_lockfile` uses this to parse the lockfile and currently it will be trying to parse a binary `bun.lockb` file as a `yarn.lock` structure.

Reply:
Fixing that up now to do the same thing as Go for `bun bun.lockb`.